rotkehlchen/chain/evm/decoding/curve/curve_cache.py

```diff
@@ -388,8 +406,19 @@ def _query_curve_data_from_chain(
         deployed_block=0,  # deployment_block is not used and the contract is dynamic
     )
     pool_count = metaregistry.call(node_inquirer=evm_inquirer, method_name='pool_count')
-    new_pools = []
+    new_pools: list[CurvePoolData] = []
+    last_notified_ts = Timestamp(0)
+    chain_name = evm_inquirer.chain_id.to_name()
     for pool_index in range(pool_count):
+        last_notified_ts = maybe_notify_cache_query_status(
+            msg_aggregator=msg_aggregator,
+            last_notified_ts=last_notified_ts,
+            protocol=CPT_CURVE,
+            chain_name=chain_name,
+            processed=0,
+            total=len(new_pools),
```

Comment:
can you add a small comment mentioning that you keep processed 0 here and increment `new_pools` and the processed get incremented in the function `add_name_here`. It was confusing even after explaining it in discord but having the other function name helped here.